### PR TITLE
CLIF: Add missing translations for classes and interfaces

### PR DESCRIPTION
**Pull request by mbtools**
Closes a long-time gap and now handles the translation of class and interface descriptions :-)

New test repos:
https://github.com/abapGit-tests/CLAS_i18n
https://github.com/abapGit-tests/INTF_i18n

@albertmink, it should be ready for AFF now (#6846)

### CODE REVIEW

**Comment by albertmink** [comment]
@mbtools that's great!
Because I'm curious, what would be the `LANGU` value for en-GB (aka 6N)?
Ref:
```XML
    <SEOCLASSTX>
     <LANGU>D</LANGU>
     <DESCRIPT>Test German</DESCRIPT>
    </SEOCLASSTX>
```

**Comment by mbtools** [comment]
The values are directly from `seoclasstx-langu`. afaik, British users logon in `EN` and get texts from `langu=E`

File: src/objects/oo/zcl_abapgit_oo_base.clas.abap [hunk 1]
```diff
@@ -261,7 +277,36 @@ CLASS zcl_abapgit_oo_base IMPLEMENTATION.
...
+      RETURN.
+    ENDIF.
+
+    " Make sure we keep main language
+    SELECT * FROM seoclasstx INTO TABLE lt_descriptions
+      WHERE clsname = is_key-clsname AND langu = iv_language
+      ORDER BY PRIMARY KEY.
+
+    LOOP AT it_descriptions ASSIGNING <ls_description> WHERE langu <> iv_language.
+      <ls_description>-clsname = is_key-clsname.
```
Comment (albertmink) [inline src/objects/oo/zcl_abapgit_oo_base.clas.abap:300]: For me it is dumping right here (when pulling https://github.com/abapGit-tests/INTF_i18n)
Screenshot 2024-07-02 at 16 08 15

Comment (mbtools) [reply src/objects/oo/zcl_abapgit_oo_base.clas.abap]: Fixed

**Review by albertmink — APPROVED** [review]
LGTM.
Tested the deserialisation of INTF with respect of SEOCLASSTX

**Pull request merged**